exercises/pangram/PangramTests.elm

```diff
@@ -0,0 +1,48 @@
+module Main (..) where
+
+import Task
+import Console
+import ElmTest exposing (..)
+import Pangram
+
+
+tests : Test
+tests =
+  suite
+    "Pangram"
+    [ test
+        "sentence empty"
+        (assertEqual
+          False
+          (Pangram.isPangram "")
```

Comment:
Is there a reason to not expose `isPanagram` and use it unqualified as you have done in your other tests?